sentry_sdk/hub.py

```diff
@@ -791,14 +791,17 @@ def trace_propagation_meta(self, span=None):
 
         sentry_trace = self.get_traceparent()
         if sentry_trace is not None:
-            meta += '<meta name="%s" content="%s">' % (
+            meta += '<meta name="%s" content="%s" />' % (
```

Comment:
Why the xhtml closing slash?

Reply:
Can this cause any problems?

Reply:
but it used to be without, so I remove again.